### ISSUE TITLE
Suggestion for next version

### ISSUE THREAD

**Issue by RomeoAlphaEkko** [Detergent13/tft-bot#129]
![image 1](https://github.com/Detergent13/tft-bot/assets/89660477/17af5ef5-fb78-4123-a760-712739e00298)

The teamplaner is finally getting "usefull" one gets an extra icon to see what one wants to buy.

pro
No tests yet but maybe one could use it so one has flexible comps and for now no changes to the code for the future.

con:
the users have to set up the teamplaner once (per account)

**Comment by MrIslamAIman** [comment]
that would be so usefull =D

**Comment by Detergent13** [comment]
Agreed! Definitely worth toying around with. Not sure how sensitive OpenCV will be with the backgrounds, or perhaps could just edit it out in the reference image. Would make the process a lot more streamlined, and moves steps to the much nicer TFT GUI. (Rather than editing files/command line)

**Comment by RomeoAlphaEkko** [comment]
We need 3 checks then pengu, 2 star, 3 star (maybe 🤔 just in case)

2 star already tested works fine

**Comment by RomeoAlphaEkko** [comment]
We would need 3 shops helmet, 2 ⭐, and maybe 3 ⭐

2 star is tested

**Comment by RomeoAlphaEkko** [comment]
2 star up seems to have a "problem" if the lobby stars up it click the star up message too
just an "unnecessary" click

**Comment by Detergent13** [comment]
I just pushed an update for 9.5 without the Team Builder feature. I got started on the implementation but didn't bother with the 2- and 3-star situations. But if you have it working feel free to commit it!

And yeah, I'll close stale issues when I get the chance.
